spec/tapioca/cli/gem_spec.rb

```diff
@@ -1116,6 +1116,94 @@ module TypedParameters; end
           RBI
         end
 
+        it "must do mixin attribution properly when include occurs in other gem" do
+          some_engine = mock_gem("some_engine", "0.0.2") do
+            write("lib/some_engine.rb", <<~RUBY)
+              require "action_controller"
+
+              module SomeEngine
+                class SomeController < ActionController::Base
+                  # This method triggers a dynamic mixin
+                  # which should be attributed to this gem and not
+                  # actionpack, even though the real `include` happens
+                  # inside actionpack
```

Comment:
Nitpick:

Do
we
really
need
that
many
lines
for
this
message?

I
though
the
limit
was
120
characters?

😂